src/lib/seam/connect/index.ts

```diff
@@ -14,6 +14,7 @@ export * from './routes/index.js'
 export * from './seam-http.js'
 export * from './seam-http-error.js'
 export * from './seam-http-multi-workspace.js'
+export * from './seam-http-request.js'
```

Comment:
Why export this in the public API?

Reply:
consumers will need to use it in some circumstances, e.g:
```typescript
function useSeamQuery<TRequest extends SeamHttpRequest<any,any>>(getRequest: (client: SeamHttp) => TRequest) {
  // ...
}
```